apps/trustlab/.env

```diff
@@ -0,0 +1,13 @@
+MIGRATIONS_DIR=./migrations
+
+SENTRY_ENVIRONMENT=local
+SENTRY_ORG=code-for-africa
+SENTRY_PROJECT=trustlab
+
+NEXT_PUBLIC_APP_URL=http://localhost:3000
+NEXT_PUBLIC_IMAGE_DOMAINS="*.codeforafrica.org"
```

Comment:
I think this is [deprecated](https://nextjs.org/docs/pages/api-reference/components/image#domains)